package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@honehealth/icons",
-  "version": "2.2.0",
+  "version": "2.2.1",
```

Since this is adding new icons, deserves a minor bump.
```suggestion
  "version": "2.3.0",
```